study.md

```diff
@@ -21,7 +21,7 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+This is dynamic in its use in that it can take on many different values while also, because of its versatility, allows users to be more specific in using an object once it has invoked a function.
```

Comment:
👍 dynamism is a key characteristic of `this` and distinction between it and prototypes